frontend/src/views/Market.vue

```diff
@@ -83,12 +83,17 @@
                 <template #above="{ character: { id } }">
                   <div class="token-price">
                     <span class="d-block text-center" v-if="nftPricesById[id]">
-                      {{ convertWeiToSkill(nftPricesById[id]) | maxDecimals(2) }} SKILL
+                      <span v-if="convertWeiToSkill(nftPricesById[id]) !== '0'">
+                        {{ convertWeiToSkill(nftPricesById[id]) | maxDecimals(2) }} SKILL
+                      </span>
                     </span>
                     <span class="d-block text-center" v-else>Loading price...</span>
                   </div>
                 </template>
 
+                <template #sold="{ character: { id } }">
+                  <div class="sold" v-if="nftPricesById[id] && convertWeiToSkill(nftPricesById[id]) === '0'"><span>SOLD</span></div>
```

Comment:
can you do text-transform: uppercase instead of uppercasing the word?

Reply:
Of course you'r the boss :)